### PR TITLE
Implements Program Counter

### PR DESCRIPTION
**Pull request by wanderer**

### CODE REVIEW

File: tests/code/solidity_add_uint256.json [hunk 1]
```diff
@@ -104,6 +104,6 @@
       "coinbase": "0x0000000000000000000000000000000000000000",
       "callData": "0x771602f7000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000420000000000000000000000000000000000000000000000000000000000000024"
     },
-    "gasUsed": 225
+    "gasUsed": 69
```
Comment (axic) [inline tests/code/solidity_add_uint256.json]: I think this wrong? 225 came from browser-solidity with that calldata.

File: index.js [hunk 2]
```diff
@@ -190,6 +190,9 @@ exports.compileEVM = function (evmCode, stackTrace) {
         }
         i += op.number
         break
+      case 'PC':
+        wasmCode = `(i64.const ${i}) ${wasmCode}`
```
Comment (axic) [inline index.js]: Why is the PC i64? We limit codesize to i32 so we could limit this to i32 as well. (Probably better to do in a separate PR)

Comment (wanderer) [inline index.js]: good point, will change
